assessment_software/src/app/dialogue-box/warnngBox-component/warningBox.component.ts

```diff
@@ -0,0 +1,11 @@
+import {Component, Input} from '@angular/core';
+
+@Component({selector:'app-warning-box',
+templateUrl:'./warningBox.component.html',
+styleUrls:['./warningBox.component.css']})
+
+export class WarningBoxComponent{
+    @Input() warning_message:string='Error';
+    @Input() warningBoxStyle:string='warning';
+    buttonStyle='warning-button';
+}
```

Comment:
Warning is not necessarily an error. if a user decides to delete an assignment or archive an assignment/course, we need to ask for confirmation. It is like messages like "are you sure you want to remove this menamen..." 

Rather than the button saying "view details", it should be "Yes"

in addition, the current implementation doesn't remove the dialogue when the cancel button is pressed. 

lastly, currently there is no way for the user to know when the "Yes" button is pressed. Please refer to the app-button-implementation that uses eventemitter to notify parent user that the button is clicked. And, when the button is clicked, it should by default hide/disappear the dialogue box.